airflow/www/views.py

```diff
@@ -87,7 +87,7 @@
     dagbag = models.DagBag
 
 
-def get_date_time_num_runs_dag_runs_form_data(request, session, dag):
+def get_date_time_num_runs_dag_runs_form_data(request, dag):
```

Comment:
Why the change to use a new session rather than re-use the one that we already have?